client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -112,9 +116,16 @@ const PluginsBrowserListElement = ( props ) => {
 		return ! isCompatiblePlugin( plugin.slug ) && ! jetpackNonAtomic;
 	} );
 
-	const shouldUpgrade = useSelector( ( state ) => shouldUpgradeCheck( state, selectedSite?.ID ) );
+	const shouldUpgradeToInstallPurchasedPlugins = useSelector( ( state ) =>
+		shouldUpgradeCheck( state, selectedSite?.ID, WPCOM_FEATURES_INSTALL_PURCHASED_PLUGINS )
+	);
+
+	const shouldUpgradeToManagePlugins = useSelector( ( state ) =>
+		shouldUpgradeCheck( state, selectedSite?.ID, WPCOM_FEATURES_MANAGE_PLUGINS )
```

Comment:
Let's use `WPCOM_FEATURES_INSTALL_PLUGINS` here to stay in line with `WPCOM_FEATURES_INSTALL_PURCHASED_PLUGINS`?